File: react/src/main/java/com/react/auth/dto/AuthenticationResponse.java
package com.react.auth.dto;

import com.react.user.entity.UserRole;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthenticationResponse {
    private String token;
    private UserRole role;
    private String refreshToken;
}

File: BaseSpringSecurity/src/main/kotlin/auth/dto/AuthenticationResponse.kt
package com.auth.payload

import com.entity.RefreshToken
import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor

data class AuthenticationResponse(
    var token: String? = null,
    var role: String? = null,
    var refreshToken: String? = null,
)